Makefile

```diff
@@ -18,14 +18,29 @@ clean-regtest:
 clean-regtest-data:
 	rm -rf data/regtest/spvnode data/regtest/wallet
 
-package: clean-package package-linux package-win package-macos
+package: clean-package package-linux package-debian package-win package-macos
 
 package-linux:
 	pkg . --targets node16-linux-x64 --out-path dist/linux
 	mkdir -p dist/linux/prebuilds
 	cp -r node_modules/leveldown/prebuilds/linux-x64 dist/linux/prebuilds
 	tar -czvf dogecoin-spv-$(VERSION)-linux-x64.tar.gz dist/linux
 
+CONTROL_FILE = dist/debian/dogecoin-spv/DEBIAN/control
+package-debian: package-linux
+	rm -rf dist/debian/
+	mkdir -p dist/debian/dogecoin-spv/DEBIAN/;mkdir -p dist/debian/dogecoin-spv/usr/bin/
+	cp -r dist/linux/* dist/debian/dogecoin-spv/usr/bin/
+	# create control file
+	touch $(CONTROL_FILE)
+	echo "Package: dogecoin-spv" >> $(CONTROL_FILE)
+	echo "Architecture: amd64" >> $(CONTROL_FILE)
+	echo "Version: $(VERSION)-1" >> $(CONTROL_FILE)
+	echo "Maintainer: Malik <malik.mlitat@gmail.com>" >> $(CONTROL_FILE)
+	echo "Description: simple spv node for Dogecoin." >> $(CONTROL_FILE)
```

Comment:
We actually have now a spv node + wallet. So in the description we could say "A simple spv wallet for Dogecoin."

Reply:
fixed